src/client/components/ReserveListings/index.js

```diff
@@ -1,5 +1,100 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Link } from '@instructure/ui-link';
+import { Spinner } from '@instructure/ui-spinner';
+import { Table } from '@instructure/ui-table';
+import { Text } from '@instructure/ui-text';
+import { TruncateText } from '@instructure/ui-truncate-text';
+import { ltikPromise } from '../../services/ltik';
 
-const ReserveListings = () => <div>hi</div>;
+const ReserveListings = () => {
+  const [entries, setEntries] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  const getReserves = () => {
+    ltikPromise.then(ltik => {
+      axios
+        .get(`/api/getreserves?ltik=${ltik}`, {
+          params: { term: '20S' },

```

Comment:
@kilometerskang This is hardcoded to use 20S. Should show all terms or a given term. See the term selector for Bruincast content at https://github.com/ucla/lti-media-resources/blob/master/src/client/components/AdminPanel/BruincastAdminListings.js#L60